<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindowHelp</class>
 <widget class="QWidget" name="GameWindowHelp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>911</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1221</width>
    <height>911</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1221</width>
    <height>911</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>-50</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gameplay</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="setUpHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="setUpImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>image goes here</string>
   </property>
  </widget>
  <widget class="QLabel" name="setUpMain">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Flagger has a total of 3 total difficulties ranging from easy to hard with a total of 36 unique flags.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="guessImage">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>300</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>image goes here</string>
   </property>
  </widget>
  <widget class="QLabel" name="hintImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>image goes here</string>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackImage">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>750</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>image goes here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
